### PR TITLE
Add shortnames to kueue APIs

### PR DESCRIPTION
**Pull request by kerthcet**
Signed-off-by: kerthcet <kerthcet@gmail.com>

#### What type of PR is this?
/kind feature

#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:
NONE

### CODE REVIEW

File: apis/kueue/v1alpha2/workload_types.go [hunk 1]
```diff
@@ -120,6 +120,7 @@ const (
 // +kubebuilder:printcolumn:name="Queue",JSONPath=".spec.queueName",type=string,description="Name of the queue this workload was submitted to"
 // +kubebuilder:printcolumn:name="Admitted by",JSONPath=".spec.admission.clusterQueue",type=string,description="Name of the ClusterQueue that admitted this workload"
 // +kubebuilder:printcolumn:name="Age",JSONPath=".metadata.creationTimestamp",type=date,description="Time this workload was created"
+//+kubebuilder:resource:shortName={wl}
```
Comment (alculquicondor) [inline apis/kueue/v1alpha2/workload_types.go:123]: for consistency

```suggestion
// +kubebuilder:resource:shortName={wl}
```

**Review by alculquicondor — COMMENTED** [review]
/approve
/hold for nit

File: apis/kueue/v1alpha2/workload_types.go (hunk 1 above)
Comment (kerthcet) [reply apis/kueue/v1alpha2/workload_types.go:123]: Absolutely.

**Comment by kerthcet** [comment]
Updated.

**Review by alculquicondor — COMMENTED** [review]
/lgtm
Thanks!

**Pull request merged**